mantle/platform/machine/aws/flight.go

```diff
@@ -16,14 +16,17 @@
 package aws
 
 import (
+	"time"
+
 	"github.com/coreos/pkg/capnslog"
 
 	"github.com/coreos/mantle/platform"
 	"github.com/coreos/mantle/platform/api/aws"
 )
 
 const (
-	Platform platform.Name = "aws"
+	Platform            platform.Name = "aws"
+	provisioningTimeout               = 3 * time.Minute
```

Comment:
I feel like these would read a little better if we added the type in here too:

```suggestion
	Platform            platform.Name = "aws"
	provisioningTimeout time.Duration = 3 * time.Minute
```